impeller/display_list/display_list_dispatcher.cc

```diff
@@ -268,9 +269,26 @@ void DisplayListDispatcher::setColorSource(
       paint_.contents = std::move(contents);
       return;
     }
+    case flutter::DlColorSourceType::kSweepGradient: {
+      const flutter::DlSweepGradientColorSource* sweepGradient =
+          source->asSweepGradient();
+      FML_DCHECK(sweepGradient);
+      auto contents = std::make_shared<SweepGradientContents>();
+      contents->SetCenterAndAngles(ToPoint(sweepGradient->center()),
+                                   Degrees(sweepGradient->start()),
+                                   Degrees(sweepGradient->end()));
```

Comment:
Like with most angles, I think these values are actually given in [radians](https://api.flutter.dev/flutter/painting/SweepGradient-class.html)?

Reply:
Looks like most of the angles coming in through the dispatcher are actually in degrees though. I haven't checked if/where these particular angles are being converted.

Reply:
The angles here are in degrees, and the conversion from radians to degrees is here :)
https://github.com/flutter/engine/blob/953d0d18c6ed9e7a48a1a46ed9e176dda0a601bc/lib/ui/painting/gradient.cc#L103-L106

Reply:
Perfect, thanks for verifying!